mne/preprocessing/bads.py

```diff
@@ -30,7 +33,14 @@ def find_outliers(X, threshold=3.0, max_iter=2):
     my_mask = np.zeros(len(X), dtype=np.bool)
     for _ in range(max_iter):
         X = np.ma.masked_array(X, my_mask)
-        this_z = np.abs(zscore(X))
+        if tail == "two":
```

Comment:
I think we usually use `tail=0 | 1 | -1` in our stats functions to indicate directionality, might as well do the same here